tests/scadnano_tests.py

```diff
@@ -919,10 +928,11 @@ def test_export_design_with_helix_group(self):
         }
 
         design = sc.Design(helices=helices, groups=groups, strands=[])

```

Comment:
For the cadnano export unit tests, let's do something similar to the "reverse" tests (of importing from cadnano), where we don't exhaustively test that the whole design looks good, but we pick a couple of small things to test, for example, the number of helices and strands, just enough to make sure that it doesn't write an empty string, for instance.

I think it would be perfectly fine to do this by simply calling the import methods to get a new scadnano Design back, and then have a few `self.assertEqual` statements related to it. (rather than manually testing the json exported from `to_cadnano_v2_json`).